WcaOnRails/app/assets/javascripts/registrations.js

```diff
@@ -38,24 +38,24 @@ onPage('registrations#index', function() {
 });
 
 function comparePaymentDate(a, b) {
-  var elemA = $(a);
-  var elemB = $(b);
-  var paidA = elemA.data("paidDate");
-  var paidB = elemB.data("paidDate");
+  const elemA = $(a);
+  const elemB = $(b);
+  const paidA = elemA.data("paidDate");
+  const paidB = elemB.data("paidDate");
   if (paidA !== "" && paidB !== "") {
     // Both have paid, compare their last payment dates
     return paidA.localeCompare(paidB);
-  } else if (paidA === "" && paidB === "") {
+  }
+  if (paidA === "" && paidB === "") {
     // None have paid, compare their registration dates
     return elemA.data("registeredAt").localeCompare(elemB.data("registeredAt"));
-  } else {
-    return paidA === "" ? 1 : -1;
   }
+  return paidA === "" ? 1 : -1;
 }
 
 function compareHtmlContent(a, b) {
-  var first = $('<p>' + a + '</p>').text().trim();
-  var second = $('<p>' + b + '</p>').text().trim();
+  const first = $('<p>' + a + '</p>').text().trim();
+  const second = $('<p>' + b + '</p>').text().trim();
```

Comment:
Seems like your linter tapped into unrelated code?

Reply:
Whoops! I meant to roll those changes back before commiting sorry! Think I've fixed it now...